linux/devdoc/async_socket_linux_requirements.md

```diff
@@ -231,9 +191,7 @@ MOCKABLE_FUNCTION(, ASYNC_SOCKET_SEND_SYNC_RESULT, async_socket_send_async, ASYN
 
   - **SRS_ASYNC_SOCKET_LINUX_11_056: [** `async_socket_send_async` shall create a context for the send where the `payload`, `on_send_complete` and `on_send_complete_context` shall be stored. **]**
 
-  - **SRS_ASYNC_SOCKET_LINUX_11_057: [** The the context shall then be added to the epoll system by calling `epoll_ctl` with `EPOLL_CTL_MOD`. **]**
-
-  - **SRS_ASYNC_SOCKET_LINUX_11_058: [** If the `epoll_ctl` call fails with `ENOENT`, `async_socket_send_async` shall call `epoll_ctl` again with `EPOLL_CTL_ADD`. **]**
+  - **SRS_ASYNC_SOCKET_LINUX_11_057: [** The context shall then be added to the completion port system by calling `completion_port_add` with `EPOLL_CTL_MOD`. **]**
 
 - **SRS_ASYNC_SOCKET_LINUX_11_059: [** If the `errno` value is `ECONNRESET`, `ENOTCONN`, or `EPIPE` shall fail and return `ASYNC_SOCKET_SEND_SYNC_ABANDONED`. **]**
```

Comment:
>ASYNC_SOCKET_SEND_SYNC_ABANDONED [](http://example.com/codeflow?start=123&length=32)

I don't think we should ever return ABADONED from any synchronous path.

Reply:
ABANDONDED means that the socket on the other end has closed